src/window.rs

```diff
@@ -43,8 +44,14 @@ pub fn draw_gui() {
     let num_dials = 5;
     let mut dials = Vec::new();
 
+    let range_size = 4000.0;
     for i in 0..num_dials {
-        let dial = Dial::new(i + 1, (i + 1) as f32 * 100.0);
+        let dial = Dial::new(
+            i + 1,
+            50.0,
+            DialRange::new(i as f32 * 200.0, i as f32 * 200.0 + range_size),
+            (i + 1).to_string().chars().next().unwrap(),
```

Comment:
lol. If it works it works

Reply:
This is just for until we read it from the configuration file haha